src/views/List.jsx

```diff
@@ -23,51 +31,26 @@ export function List({ data, loading }) {
 		return shoppingListArr.length === 0 ? true : false;
 	};
 
+	/* This function will return:
+	- True if the updated urgency list has more than one item
+	- False if the list has one item or less*/
+	const checkForMoreThanOneItem = () => {
```

Comment:
This is an interesting design choice - curious to know why you chose to conditionally render the search bar?

Reply:
Good question, when we last met as a group we agreed that it made sense for the search bar to appear when there was more than one item in a list rather than in the scenario that only one item in the list existed. But let us know if you think this doesn't make sense stylistically! We were looking at how other TCL app's were formatting their search inputs.

Reply:
It's totally fine to do that. I'm not a designer by any means, but my thought here was that you would want a user to know the features available to them on any given screen. Even if they probably won't use the search bar if there is just one item, they'll know it's there for future use. But again, not a problem at all! Since the team decided on this, go with what everyone agreed on.